docs/utilities.rst

```diff
@@ -3,20 +3,40 @@ Utilities
 ===================
 
 
+Solarman RTU Proxy
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+
+    user@host:~ $ solarman-rtu-proxy -h
+    usage: solarman rtu proxy [-h] [-b BIND] [-p PORT] -l LOGGER -s SERIAL
+
+    A Modbus RTU over TCP Proxy for Solarman loggers
+
+    options:
+      -h, --help           show this help message and exit
+      -b, --bind BIND      The address to listen on
+      -p, --port PORT      The TCP port to listen on
+      -l, --logger LOGGER  The IP address of the logger
+      -s, --serial SERIAL  The serial number of the logger
+
 Solarman Scan Utility
 ^^^^^^^^^^^^^^^^^^^^^
 
-.. literalinclude:: ../utils/solarman_scan.py
-   :linenos:
+.. code-block:: console
+
+    user@host:~ $ solarman-scan
 
 Solarman Unicast Scanner
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. literalinclude:: ../utils/solarman_uni_scan.py
-    :lineos:
+.. code-block:: console
+
+    user@host:~ $ solarman-uni-scan wlan0
 
 Solarman Decoder
 ^^^^^^^^^^^^^^^^
 
-.. literalinclude:: ../utils/solarman_decoder.py
-   :linenos:
+.. code-block:: console
+
+    user@host:~ $ solarman-decoder
```

Comment:
What about <modbus frame hex>  here i.e. `user@host:~ $ solarman-decoder <modbus frame hex>`

Reply:
I decided to also leverage argparse here for a nicer help text